corehq/util/workbook_json/excel.py

```diff
@@ -162,7 +162,7 @@ def __init__(self, worksheet, title=None):
                 break
             else:
                 width += 1
-        self.worksheet.calculate_dimension(force=True)
+        self.worksheet.calculate_dimension()
```

Comment:
Does this "just work" now? is there a test for whatever `force=True` was meant to fix?

Reply:
Added back ```read_only``` in place of ```use_iterators```, which then means that ```calculate_dimension``` expects ```force=True```.  (has test coverage)